lib/ExpensiMark.js

```diff
@@ -628,4 +612,25 @@ export default class ExpensiMark {
         }
         return replacedText;
     }
+
+    /**
+     * Handle whitespace for the content of blockquote if the text matches the regex or else just return the original text
+     *
+     * @param {RegExp} regex
+     * @param {String} textToCheck
+     * @param {Function} replacement
+     *
+     * @returns {String}
+     */
+    formatWhitespaceForQuote(regex, textToCheck, replacement) {
```

Comment:
I think the function name and doc comment can be improved. This feels to me like `formatTextForQuote` as it's not only for whitespace.

Reply:
updated